apps/sms/views/shared/style/composer.css

```diff
@@ -4,14 +4,22 @@
 
   height: auto; /* overrides building blocks */
   min-height: 4rem;
-
   background-color: #fff;
 }
 
-/* Overwrite. No more fixed height */
-form[role="search"] p input,
-form[role="search"] p textarea,
-form[role="search"] button[type="submit"] + p > textarea {
+/* 
+ * https://bugzilla.mozilla.org/show_bug.cgi?id=1055357
+ * Instead of sticking to id selector I added a class to compose form,
+ * that holds styles which were related before to form[role="search"]
+ */
+
+.messages-compose {
+  z-index: 3;
+}
+
+.messages-compose p input,
+.messages-compose p textarea,
+.messages-compose button[type="submit"] + p > textarea {
   height: auto;
 }
```

Comment:
actually I think this rule is useless -- we don't use a input or a textarea in the "composer" part for a very long time